### PR TITLE
Added Imperial garbage dump

### PR DESCRIPTION
**Pull request by Lineth**
so the empire doesn't presumably offload it's rubbish onto other worlds (lore accurate though that might be)

### CODE REVIEW

File: dat/spob/bilgewater.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  <exterior>uninhabited-rocky-yellow-04.webp</exterior>
+ </GFX>
+ <general>
+  <class>Z</class>
+  <population>0</population>
+  <hide>1.000000</hide>
+  <services>
+   <land/>
+  </services>
+  <description>This world was thoroughly unpleasant even before it became the Empire's official garbage dump. Technonically hyperactive, bombarded by asteroids and solar radiation, and prone to devastating acid rain, it did everything in it's power to dissuade colonization. It was found to even lack enough rare minerals to justify a penal mining colony by even the most callous Imperial bureaucrat. However, the violence of the surface did make it convenient for legally dumping the Empire's collective effluence here and forgetting about it.</description>
```
Comment (TheGiraffe3) [inline dat/spob/bilgewater.xml:15]:
```suggestion
  <description>This world was thoroughly unpleasant even before it became the Empire's official garbage dump. Technically hyperactive, bombarded by asteroids and solar radiation, and prone to devastating acid rain, it did everything in its power to dissuade colonization. It was found to even lack enough rare minerals to justify a penal mining colony by even the most callous Imperial bureaucrat. However, the violence of the surface did make it convenient for legally dumping the Empire's collective effluence here and forgetting about it.</description>
```

**Review by TheGiraffe3 — APPROVED** [review]
Description looks alright.

**Comment by Lineth** [comment]
> Description looks alright.

Should note  that the first ward was misspelled, but its not supposed to be technically. I was trying to type tectonically, (its a perfectly cromulent word), in reference to earthquakes.

**Comment by bobbens** [comment]
Looks good, thanks! I'll merge post-0.12.0.

**Pull request merged**